src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -19,8 +19,20 @@ import toCommaSeparatedList from 'utils/toCommaSeparatedList';
 import {sharedStyles} from 'theme';
 import createOgUrl from 'utils/createOgUrl';
 
+type Props = {
+  authors: Array<string>,
+  createLink: Function,
+  date?: string,
+  enableScrollSync?: boolean,
+  ogDescription: string,
+  location: Object,
```

Comment:
Should be the `Location` type: https://github.com/facebook/flow/blob/v0.56.0/lib/bom.js#L289